spec/plugins/data_table/server_side.js

```diff
@@ -433,4 +434,57 @@ describe("DataTable Plugin", function() {
     });
   });
 
+  describe("filtering", function() {
+    beforeEach(function() {
+      table = new app.Views.T({ collection : collection });
+      table.render();
+      jasmine.Ajax.requests.mostRecent().response(mockResponse.get());
+    });
+
+    it("should have filtering info for filterable column in column manager", function() {
+      var cg = table._columnManager._configGenerator;
+      var numCols = 0;
+      var numFilters = 0;
+      for (var i = 0; i < cg.columnsConfig.length; i++) {
+        var col = cg.columnsConfig[i];
+        //console.log(JSON.stringify(col));
+        expect(col).not.toEqual(null);
+        if (col) {
+          numCols++;
+          if (col.filter)
+            numFilters++;
+        }
+      }
+      //console.log("Number of columns found in config: "+numCols);
+      //console.log("Number of columns with filters:  "+numFilters);
+      expect(numCols).toEqual(4);
+      expect(numFilters).toEqual(2);
```

Comment:
We should also test to make sure the types are actually getting put into the ColumnManager filter object/settings. I'd also include some types that aren't free form input text.